index.bs

```diff
@@ -3284,13 +3288,55 @@ The following rules will apply when calling these methods:
 Note: {{AudioParam}} attributes are read only, with the exception
 of the {{AudioParam/value}} attribute.
 
+The automation rate of an {{AudioParam}} can be selected setting the
+{{AudioParam/automationRate}} attribute with one of the following
+values.  However, some {{AudioParam}}s have constraints on whether the
+automation rate can be changed.
+
+<pre class="idl">
+[Exposed=Window]
```

Comment:
Do we need `Exposed` for enum?

Reply:
Oops. Probably too much cut'n'paste.  I don't think it's needed; we don't use it anywhere else.